index.html

```diff
@@ -0,0 +1,151 @@
+      <div class="icons">
+        <div class="icon">
+          <i class="fas fa-search"></i><br /><span class="h2">Search Places</span>
+        </div>
+        <div class="icon icon2">
+          <i class="far fa-check-circle"></i><br /><span class="h2">Select Places</span>
+        </div>
+        <div class="icon">
+          <i class="far fa-thumbs-up"></i><br /><span class="h2">Book & Enjoy</span>
+        </div>
+        <div class="line"></div>
+      </div>
+    </section>
+    <section class="book-all" id="book-all">
+      <div class="book book1">
+        <div class="container">
+          <div class="row">
+            <div class="col-12">
+              <div class="booknow">
+                <h6 class="h6">Spain</h6>
+                <h5 class="h5">Grand Spain Madrid</h5>
+                <p class="p">
+                  But I must explain to you how all this mistaken idea of account to denouncing
+                  pleasure and praising pain was account born and I account will give you a complete
+                  account of the system
+                </p>
+                <button class="button">book now</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="book book2">
+        <div class="container">
+          <div class="row">
+            <div class="col-12 d-flex justify-content-end">
+              <div class="booknow">
+                <h6 class="h6">Spain</h6>
+                <h5 class="h5">Grand Spain Madrid</h5>
+                <p class="p">
+                  But I must explain to you how all this mistaken idea of account to denouncing
+                  pleasure and praising pain was account born and I account will give you a complete
+                  account of the system
+                </p>
+                <button class="button">book now</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="tourist-review">
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h2 class="h2">What Our Tourists Say</h2>
+            <div class="review">
+              <img src="img/ReviewPhoto.jpg" alt="man" />
+              <div class="review-text">
+                <h5 class="h5">Sam Sumon</h5>
+                <p class="p">
+                  But I must explain to you how all this mistaken idea of account to
+                  denouncinpleasure and praising pain But I must explain to you how all this
+                  mistaken idea of account to nai denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the system was account born
+                  and I account will give you a nai the complete account of the system
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
```suggestion
        <h2 class="h2">What Our Tourists Say</h2>
        <div class="review">
          <div class="row g-0 align-items-stretch">
            <div class="col-4">
              <img src="img/ReviewPhoto.jpg" alt="man" />
            </div>
            <div class="col-8">
              <blockquote class="review-text">
                <cite class="h5">Sam Sumon</cite>
                <p class="p">
                  But I must explain to you how all this mistaken idea of account to
                  denouncinpleasure and praising pain But I must explain to you how all this
                  mistaken idea of account to nai denouncing pleasure and praising pain was account
                  born and I account will give you a complete account of the system was account born
                  and I account will give you a nai the complete account of the system
                </p>
              </blockquote>
            </div>
          </div>
        </div> 
```
так будет правильнее